### PR TITLE
EG-2239 (DOC-1657): Refactor Corda OS Tutorials

### PR DESCRIPTION
**Pull request by taylorharvey-r3**
This PR covers the following tickets:

- DOC-1706
- DOC-188
- DOC-451
- DOC-239
- DOC-557
- DOC-518

This mostly involves updates and corrections to the Hello World tutorials and Running a sample CorDapp tutorial. 

All code examples and commands have been tested (and updated where needed) and the user can run these tutorials without issue following these instructions. 

Work under this epic (formerly EG-2239, now DOC-1657) will continue in a separate PR.

### CODE REVIEW

File: content/en/docs/corda-os/4.7/hello-world-running.md [hunk 1]
```diff
@@ -135,12 +135,23 @@ More about SSH and how to connect can be found on the [Node shell](shell.md) pag
 You want to create an IOU of 99 with `PartyB`. To start the `IOUFlow`, type the following syntax:
 
 ```bash
-start IOUFlow iouValue: 99, otherParty: "O=PartyB,L=New York,C=US"
+start IOUFlow iouMessage: 99, otherParty: "O=PartyB,L=New York,C=US"
 ```
 
 This single command will cause `PartyA` and `PartyB` to automatically agree an IOU. This is one of the great advantages of
 the flow framework - it allows you to reduce complex negotiation and update processes into a single function call.
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-os/4.7/hello-world-running.md:143]:
```suggestion


```

**Comment by taylorharvey-r3** [comment]
@AnnaK-R3 You can see the changes here on my preview site: https://tolkien.corda.net/docs/corda-os/4.7/hello-world-introduction.html

File: content/en/docs/corda-os/4.7/hello-world-running.md [hunk 2]
```diff
@@ -135,12 +135,23 @@ More about SSH and how to connect can be found on the [Node shell](shell.md) pag
...
 
+Starting this flow will return the following:
+```bash
+✅   Starting
+         Requesting signature by notary service
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/hello-world-running.md:147]:
```suggestion
         Requesting signature by Notary Service
```

File: content/en/docs/corda-os/4.7/hello-world-running.md [hunk 3]
```diff
@@ -135,12 +135,23 @@ More about SSH and how to connect can be found on the [Node shell](shell.md) pag
...
+             Requesting signature by Notary service
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/hello-world-running.md:148]:
```suggestion
             Requesting signature by Notary Service
```

File: content/en/docs/corda-os/4.7/hello-world-running.md [hunk 4]
```diff
@@ -135,12 +135,23 @@ More about SSH and how to connect can be found on the [Node shell](shell.md) pag
...
+             Validating response from Notary service
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/hello-world-running.md:149]:
```suggestion
             Validating response from Notary Service
```

File: content/en/docs/corda-os/4.7/hello-world-template.md [hunk 5]
```diff
@@ -38,30 +38,47 @@ Open a terminal window in the directory where you want to download the CorDapp t
...
 {{% tab name="java" %}}
 ```java
-git clone https://github.com/corda/cordapp-template-java.git ; cd cordapp-template-java
+git clone https://github.com/corda/cordapp-template-java.git
 ```
 {{% /tab %}}
 
 {{< /tabs >}}
 
+Once you have cloned the repository you wish to use, navigate to the correct sub-directory:
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/hello-world-template.md:53]:
```suggestion
Once you have cloned the repository you wish to use, navigate to the correct subdirectory:
```

File: content/en/docs/corda-os/4.7/tut-two-party-contract.md [hunk 6]
```diff
@@ -195,7 +194,9 @@ public class IOUContract implements Contract {
 
 {{< /tabs >}}
 
-If you’re following along in Java, you’ll also need to rename `TemplateContract.java` to `IOUContract.java`.
+{{< note >}}
+Be sure to update the `@BelongsToContract` annotation in the state definition ([Writing the state](hello-world-state.md)) to specify `IOUContract` class.
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/tut-two-party-contract.md:198]:
```suggestion
Ensure to update the `@BelongsToContract` annotation in the state definition ([Writing the state](hello-world-state.md)) to specify `IOUContract` class.
```

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 7]
```diff
@@ -294,11 +294,11 @@ The `cordapp-example` folder is structured as follows:
...
-* The **root directory** contains some gradle files, a README, a LICENSE and a TRADEMARK statement
-* **clients** contains the source code for Spring Boot integration
-* **config** contains the log4j2 configuration
-* **contracts** and **workflows** contain the source code for the sample CorDapp written in Java
-* **gradle** contains the gradle wrapper, which allows the use of Gradle without installing it yourself and worrying about which version is required
+* The **root directory** contains some gradle files, a README, a LICENSE and a TRADEMARK statement.
+* **clients** contains the source code for Spring Boot integration.
+* **config** contains the log4j2 configuration.
+* **contracts** and **workflows** contain the source code for the sample CorDapp written in Java.
+* **gradle** contains the gradle wrapper, which allows the use of Gradle without installing it yourself and worrying about which version is required.
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:301]:
```suggestion
* **gradle** contains the Gradle Wrapper, which allows the use of Gradle without installing it yourself and worrying about which version is required.
```

File: content/en/docs/corda-os/4.7/tutorials-index.md [hunk 8]
```diff
@@ -30,7 +30,7 @@ Complete each of the tutorial topics in sequence to learn how to extend the [Jav
 
 ## Core tutorials
 
-Once you've worked through the Hello World! tutorials and are ready to dig a bit deeper, the [Core tutorials](core-tutorials-index.md) cover all of the key skills you will need to be able to develop CorDapps.
+Once you've worked through the Hello, World! tutorials and are ready to dig a bit deeper, the [Core tutorials](core-tutorials-index.md) cover all of the key skills you will need to be able to develop CorDapps.
```
Comment (AnnaK-R3) [inline content/en/docs/corda-os/4.7/tutorials-index.md:33]:
```suggestion
Once you've worked through the Hello, World! tutorials and are ready to dig a bit deeper, the [Core tutorials](core-tutorials-index.md) cover all of the key skills you need to develop CorDapps.
```

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**